.ci/steps.win.yml

```diff
@@ -0,0 +1,7 @@
+steps:
```

Comment:
I'd refer `ci` directory to `.ci`

Reply:
I thought about it, but won't it be confusing for users? It'll look like one of the available modules.

Reply:
It's still listed in github, which is where 99% of people will ever see this repo's source.

Reply:
ok good point